src/Toml/Bi/Combinators.hs

```diff
@@ -144,6 +145,9 @@ text = match _Text
 string :: Key -> TomlCodec String
 string = match _String
 
+date :: Key -> TomlCodec DateTime
+date = match _Date
```

Comment:
Actually, it's not that useful to have `date` combinator because you can't do match with `DateTime` data type from `tomland` library. We need to have 4 combinators for types from `time` library:

1. `zonedTime`
2. `localTime`
3. `day`
4. `timeOfDay`